data/en/1_0005.json

```diff
@@ -433,7 +433,16 @@
                                 "q_code": "60",
                                 "type": "Currency",
                                 "currency": "GBP",
-                                "decimal_places": 2
+                                "decimal_places": 2,
+                                "max_value": {
+                                    "answer_id": "weekly-pay-gross-pay-answer"
+                                },
+                                "validation": {
+                                    "messages": {
+                                        "NUMBER_TOO_LARGE": "Enter a number less than or equal to £%(max)s."
```

Comment:
If we are formatting as currency, can we revert to using the default error messages?

Reply:
@jonnyshaw89 but the default error messages don't contain the previous figure in them

Reply:
don't they?
https://github.com/ONSdigital/eq-survey-runner/blob/master/app/validation/error_messages.py#L10